docs: Fix lots of typos and grammar errors

mowies opened the pull request:
This PR fixes lots of typos and grammar errors in the tutorials

jetzlstorfer commented on site/tutorials/snippets/07/install/authCLI-istio.md:
```diff
@@ -31,4 +31,4 @@ If you want, you can go ahead and take a look at the Keptn API by navigating to
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
the preview does not render this file, is it available?

jetzlstorfer commented on site/tutorials/snippets/07/install/open-bridge.md:
```diff
@@ -22,4 +22,4 @@ Positive
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
same as previous comment

jetzlstorfer commented on site/tutorials/snippets/08/install/authCLI-istio.md:
```diff
@@ -31,4 +31,4 @@ If you want, you can go ahead and take a look at the Keptn API by navigating to
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
file is not rendered correctly

jetzlstorfer commented on site/tutorials/snippets/08/install/open-bridge.md:
```diff
@@ -22,4 +22,4 @@ Positive
 
-![api](./assets/keptn-api.png)
\ No newline at end of file
+![api](./assets/keptn-api.png)
```
file not rendered correctly

jetzlstorfer reviewed (changes requested):
please have a look at one file/image that is not rendered correctly

jetzlstorfer commented:
I realized that in the generated preview, the image is available https://deploy-preview-181--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-dynatrace-08/index.html?index=..%2F..index#6 

However, the screenshot shows an old API version. @mowies could you please update the image for the 0.8.x versions? (we can keep the old image for the old versions of course)

mowies commented:
@jetzlstorfer I added a new screenshot of API version 0.8.7 and replaced it in the correct files. You can check it [here](https://deploy-preview-181--keptn-tutorials.netlify.app/tutorials/keptn-full-tour-dynatrace-08/index.html?index=..%2F..index#6)

jetzlstorfer reviewed (approved):
lgtm
